app/utils.py

```diff
@@ -367,6 +367,33 @@ def validate_cache_seconds(string):
     return 1.0
 
 
+# ==== Fuzzification functions =================================================
+
+# The range should be no more specific than a five year period.
+MIN_AGE_RANGE = 5
+
+def fuzzify_age(value):
+    if not value:
+        return None
+    parse = AGE_RE.match(value)
+    if not parse:
+        raise ValueError('Invalid age value.')
```

Comment:
Drop-by comment: Just curious, do we have validation somewhere to verify that the age is in a valid format, for both records from the web UI and API? I'm wondering if it is safe to just thrown an exception here.

Reply:
Oh, good point. We do have validation for input through the web UI, but not the API afaict. Changed it to return None for stuff it can't parse.

Reply:
Just curious, do you know how None is rendered in the template? I just want to make sure that we don't render "Age: None" literally.

Reply:
I changed the "if person.age" check to check fuzzified_age instead, so it won't do anything if it's None.